app/Main.hs

```diff
@@ -11,8 +11,11 @@ import Polysemy
 import Servant.Client (BaseUrl(..), Scheme(Http))
```

Comment:
The file `test.toml` should be deleted (or rather, replaced with/renamed to `config.toml`)

Reply:
Done